### PR TITLE
refactor(orc8r): [orc8r][ci] Add git status output to slack failing notification

### LINKED ISSUE #7992: Integrate git status in orc8r cloud test slack notifications
In the new or8r github actions test include the git status message of the   insync-checkin job.

### PR DESCRIPTION
**Pull request by quentinDERORY**
Signed-off-by: Quentin, Derory <15911421+quentinDERORY@users.noreply.github.com>

## Summary

Add git status output to insync-checkin slack failure notification

## Test Plan

Tested in my local repo

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by mattymo — APPROVED** [review]

File: .github/workflows/cloud-workflow.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
             python3 build.py --generate
             sudo chown -R $USER $MAGMA_ROOT/*
             git add .
-            git status
+            echo GIT_STATUS=`git status` >> $GITHUB_ENV
```
Comment (hcgatewood) [inline .github/workflows/cloud-workflow.yml]: Can you both save it and print it here (that way we can view it both from the Slack notification and in the job

Comment (hcgatewood) [reply .github/workflows/cloud-workflow.yml]: Also, nit for uniformity:
```suggestion
            echo GIT_STATUS=$(git status) >> $GITHUB_ENV
```

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Thanks for this @quentinDERORY! One small request, then will be ready

File: .github/workflows/cloud-workflow.yml [hunk 2]
```diff
@@ -42,7 +42,8 @@ jobs:
             python3 build.py --generate
             sudo chown -R $USER $MAGMA_ROOT/*
             git add .
-            echo GIT_STATUS=`git status` >> $GITHUB_ENV
+            echo GIT_STATUS="$(git status)" >> $GITHUB_ENV
+            echo $GIT_STATUS
```
Comment (hcgatewood) [inline .github/workflows/cloud-workflow.yml]: Does this work? Nice

**Review by hcgatewood — APPROVED** [review]
💯

File: .github/workflows/cloud-workflow.yml (hunk 2 above)
Comment (quentinDERORY) [reply .github/workflows/cloud-workflow.yml]: Now yes !

**Pull request merged**